cumulus_library/builders/counts.py

```diff
@@ -14,37 +13,28 @@ class CountsBuilder(BaseTableBuilder):
     """Extends BaseTableBuilder for counts-related use cases"""
 
     def __init__(self, study_prefix: str | None = None):
-        if study_prefix is None:
-            # This slightly wonky approach will give us the path of the
-            # directory of a class extending the CountsBuilder
-            study_path = Path(sys.modules[self.__module__].__file__).parent
```

Comment:
I moved this whole chunk of logic to `prepare_queries` and just used the existing manifest that it sends down, rather than trying to re-discover the manifest and its options.